test/cri-containerd/container_virtual_device_test.go

```diff
@@ -39,11 +46,51 @@ func verifyGPUIsPresent(t *testing.T, client runtime.RuntimeServiceClient, ctx c
 	}
 }
 
+// verifyGPUIsPresent is a helper function that runs a command in the container
+// to verify the existence of a GPU and fails the running test is none are found
+func verifyGPUIsPresentWCOW(t *testing.T, client runtime.RuntimeServiceClient, ctx context.Context, containerID string) {
+	execReq := makeGPUExecCommand("windows", containerID)
+	response := execSync(t, client, ctx, execReq)
+	if len(response.Stderr) != 0 {
+		t.Fatalf("expected to see no error, instead saw %s", string(response.Stderr))
+	}
+	out := string(response.Stdout)
+	devices := strings.Split(out, ",")
+	if len(devices) == 0 {
+		t.Fatal("expected to see devices on container, none found")
+	}
+	for _, d := range devices {
+		if strings.HasPrefix(d, gpuWin32InstanceIDPrefix) {
+			// found the target, return
+			return
+		}
+	}
+	t.Fatal("expected to see a GPU device, none present")
+}
+
+func verifyGPUIsNotPresentWCOW(t *testing.T, client runtime.RuntimeServiceClient, ctx context.Context, containerID string) {
```

Comment:
This has a bunch of duplication with `verifyGPUIsPresentWCOW`. We should instead make a single function `isGPUPresentWCOW` that returns a bool.

Reply:
Done